README.md

```diff
@@ -9,10 +9,10 @@ Rust based CLI utility for catalyst operations.
 Build with `cargo build` and run, or run with `cargo run -- PARAMS` as per the examples.
 
 ```shell
-catalyst-toolbox-cli 0.1.0
+catalyst-toolbox 0.1.0
 
 USAGE:
-    catalyst-toolbox-cli.exe [FLAGS] [SUBCOMMAND]
+    catalyst-toolbox.exe [FLAGS] [SUBCOMMAND]
```

Comment:
This looks Windows-specific :frowning_man:

Reply:
Actually yes. I think I cleaned it at some point, but probably left some. Thanks!